compiler/codegen/Relocation.cpp

```diff
@@ -391,7 +391,7 @@ const char *TR::ExternalRelocation::_externalRelocationTargetKindNames[TR_NumExt
    "TR_RamMethodSequenceReg (23)",
    "TR_VerifyClassObjectForAlloc (24)",
    "TR_ConstantPoolOrderedPair (25)",
-   "TR_AbsoluteMethodAddressOrderedPair (36)",
+   "TR_AbsoluteMethodAddressOrderedPair (26)",
```

Comment:
good catch :)

Reply:
Took 30 minutes to figure out why the logs weren't printing out "TR_DebugCounter (58)"...